third_party/move/move-core/types/src/vm_status.rs

```diff
@@ -43,6 +43,12 @@ pub static EXECUTION_STATUS_MIN_CODE: u64 = 4000;
 /// The maximum status code for runtim statuses
 pub static EXECUTION_STATUS_MAX_CODE: u64 = 4999;
 
+/// When `Addition` operation overflows the `limit`.
+pub const EADD_OVERFLOW: u64 = 0x02_0001;
```

Comment:
Aggregators are not part of Move, we shouldn't put it here. Why not use the codes from `aggregatr_extension.rs`?

Reply:
Removed it. I shifted the `is_aggregator_error()` check inline in `aptos_vm.rs`.